.travis.yml

```diff
@@ -29,7 +29,7 @@ git:
   submodules: false
 
 before_install:
-  - .travis/prepare.sh
+  - source .travis/prepare.sh
```

Comment:
I assume this has been done to allow variables to pass between the prepare and build scripts.  I don't like this approach at all. Very brittle and difficult to follow.

Reply:
After uploading the patch I realized that this is not the right way to do this and started work on a better approach where the vars are dumped to a file which can sourced by the build script. Patch coming up shortly.